lib/modules/subredditManager.js

```diff
@@ -871,14 +876,20 @@ addModule('subredditManager', {
 			/* this probably isn't the best way to give the option, since the mechanic is drag/drop for other stuff..  but it's much easier for now... */
 			$(this.staticShortCutsContainer).html('');
 			var specialButtonSelected = {};
-			var subLower = (RESUtils.currentSubreddit()) ? RESUtils.currentSubreddit().toLowerCase() : 'home';
-			specialButtonSelected[subLower] = 'RESShortcutsCurrentSub';
+			var subLower = 'home'; // default
+			if (RESUtils.currentSubreddit()) {
+				subLower = RESUtils.currentSubreddit().toLowerCase();
+			} else if ($('html').is('.res-me-f-all')) {
```

Comment:
Hmm, users can disable the module that adds that class. Might should add a url regex in utils.js to check for this case.